### PR TITLE
fix(tool/judgers.md): wrong usage in building lemon

### PR DESCRIPTION
**Pull request by CoelacanthusHex**
1.  we can pass parameter into `./make` instead of using `sed` to modify the file,
    `-jn` can be passed although `-j n` can't.
2.  we can install the executable file into PATH instead of throwing it into home dir.

the result of git blame: https://github.com/OI-wiki/OI-wiki/commit/47acc594f2d94423d319a7b1ee83546ad29552a6

### CODE REVIEW

**Review by Enter-tainer — APPROVED** [review]
thx

File: docs/tools/judgers.md [hunk 1]
```diff
@@ -174,13 +174,11 @@ Ubuntu：
 ```bash
 sudo apt update
 sudo apt install qt5-default build-essential git -y
-git clone --depth=1 http://github.com/menci/lemon.git
+git clone --depth=1 https://github.com/Menci/Lemon.git
```
Comment (Xeonacid) [inline docs/tools/judgers.md:177]: 改成LemonLime？ :thinking:

Comment (CoelacanthusHex) [reply docs/tools/judgers.md:177]: LemonLime 在用 CMake, 而且做了一些其他的修改，感觉会显得复杂许多（尤其是要处理 Qt 版本问题）
LemonLime 的 BUILD 手册： https://github.com/Project-LemonLime/Project_LemonLime/blob/master/BUILD.md

Comment (Xeonacid) [reply docs/tools/judgers.md:177]: 把这段全删掉，就留个这link，也没问题？

Comment (CoelacanthusHex) [reply docs/tools/judgers.md:177]: 我补上吧

**Review by Xeonacid — APPROVED** [review]
thx！

**Review by AzusaCat1111 — APPROVED** [review]

**Review by StudyingFather — APPROVED** [review]

**Comment by CoelacanthusHex** [comment]
three approved, no merge（逃

**Pull request merged**